docs/components/base/wheeled.md

```diff
@@ -8,11 +8,11 @@ images: ["/icons/components/base.svg"]
 tags: ["base", "components"]
 aliases:
   - "/components/base/wheeled/"
-component_description: "Mobile wheeled robot."
+component_description: "Supports mobile wheeled robotic bases with motors on both sides  for differential steering."
```

Comment:
```suggestion
component_description: "Supports mobile wheeled robotic bases with motors on both sides for differential steering."
```
[nit]

Reply:
I don't see a change there but happy to apply

Reply:
Extra space